src/air_energies.jl

```diff
@@ -230,6 +230,23 @@ and total specific humidity, given
     )
 end
 
+"""
+    specific_dry_enthalpy_vapor(param_set, T)
+
+The specific dry enthalpy of vapor, given
+
+ - `param_set` an `AbstractParameterSet`, see the [`Thermodynamics`](@ref) for more details
+ - `T` temperature
+"""
+@inline function specific_dry_enthalpy_vapor(
```

Comment:
@tapios Is this correct? I don't think I heard this definition before. Maybe I will define enthalpy and subtract Lv,0 in surface flux.